mix.exs

```diff
@@ -46,4 +46,4 @@ defmodule Nerves.Firmware.HTTP.Mixfile do
         { :exjsx, "~> 3.2.0" },
         { :ex_doc, "~> 0.11", only: :dev} ]
     end
-end
\ No newline at end of file
+end
```

Comment:
the newline is removed here, not sure how to remove the file from the commit now that everything is squashed